test/services/supplementary-billing/fetch-invoice-account-numbers.service.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseHelper = require('../../support/helpers/database.helper.js')
+const InvoiceAccountHelper = require('../../support/helpers/crm-v2/invoice-account.helper.js')
+
+// Thing under test
+const FetchInvoiceAccountNumbersService = require('../../../app/services/supplementary-billing/fetch-invoice-account-numbers.service.js')
+
+describe('Fetch Invoice Account Numbers service', () => {
+  beforeEach(async () => {
+    await DatabaseHelper.clean()
+  })
+
+  describe('when the service is called with an array of charge versions', () => {
+    let expectedResult
+    let invoiceAccounts
+
+    beforeEach(async () => {
+      // We create three invoice accounts but we will only be fetching the first two
+      invoiceAccounts = await Promise.all([
+        await InvoiceAccountHelper.add(),
```

Comment:
Don't think we need to await the `InvoiceAccountHelper` if using the `Promise.all`